include/abb_librws/rws_rapid.h

```diff
@@ -371,6 +372,243 @@ struct RAPIDRecord : public RAPIDSymbolDataAbstract
   std::vector<RAPIDSymbolDataAbstract*> components_;
 };
 
+/**
+ * \brief A struct, for representing the data of a RAPID array of atomic type.
+ */
+template<typename T>
+struct RAPIDArray : public RAPIDSymbolDataAbstract
+{
+public:
+  using value_type = T;
+
+  /**
+   * \brief A constructor.
+   *
+   * \param array_type_name specifying the name of the RAPID type of the contained items
+   * \param array_size specifying the size of the array to allocate
+   * \param value initiale value of all the item of the array
+   */
+  RAPIDArray(
+    const std::string& array_type_name, 
+    const size_t array_size,
+    const value_type& value = value_type()
+  )
+  :   array_type_name_ { array_type_name }
+  ,   container_ {std::vector<value_type>(array_size, value)}
```

Comment:
Can be just `container_ (array_size, value)`, no need to create a temporary `std::vector`.

Reply:
Removed in later commit